irohad/validation/stateless_validator.cpp

```diff
@@ -45,11 +46,11 @@ namespace iroha {
 
       // tx is not sent from future
       {
-        if (now.count() < transaction.created_ts) {
+        if (now.count() < transaction.created_ts + 300 && transaction.created_ts < now.count() - 1440*60000) {
```

Comment:
This is ugly, but I added it in order to state the fact that time validation should be a range that is slightly fuzzy, because clocks can be different on different peers. We probably need a network time service to provide a good solution.

Also note that putting time validation using now.count() limits this stateless validation to only NEW transactions. It cannot be used to validate a chain in syncing, so this this should be moved elsewhere (we have to do stateless validation and stateful validation at each block when syncing).